src/lib/clients/phony.ts

```diff
@@ -1,21 +1,69 @@
-import { BaseOptions } from "../../types/generic";
-import { Client } from "../../types/generic";
+import { BaseOptions, HandlerBody } from '../../types/generic';
+import { SayArguments } from "@slack/bolt";
+import { KeeperSlackMiddleware } from "../../types/clients/keeper"
+import { CommandClient, Command } from '../../types/generic';
 import { Logger } from '../../types/logger'
+import { EventEmitter } from 'events';
+import * as net from 'net';
 
-export default class PhonyClient implements Client {
+export default class PhonyClient extends EventEmitter implements CommandClient<PhonyClient> {
```

Comment:
Do we need to still extend `EventEmitter` since we're creating a TCP server?

Reply:
We sure don't! Good catch